### PR TITLE
Changes name of "usa" holiday to remove ambiguity.

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
"Independence Day" is now "US Independence Day"

## Why It's Good For The Game
There are dozens of independence days around the world, the current name is too ambiguous.

## Changelog
:cl:
tweak: The holiday observed on every 4th of July is now called "US Independence Day" instead of just "Independence Day".
/:cl:

### CODE REVIEW

**Comment by MrStonedOne** [comment]
> There are dozens of independence days around the world,

But only one that matters!

**Comment by interestingusernam3** [comment]
>But only one that matters!

Yes, and it's Waitangi Day.

**Comment by D4C-420** [comment]
I thought it was a movie reference

**Comment by optimumtact** [comment]
We should remove this day and replace it with Waitangi day

**Comment by Cyberboss** [comment]
I'd rather keep it as is for the movie reference.

Or remove it entirely.

**Comment by Pumpkinoe** [comment]
Waitangi day now or bust

File: code/modules/holiday/holidays.dm [hunk 1]
```diff
@@ -253,7 +253,7 @@
 	return pick("Ayy","Truth","Tsoukalos","Mulder","Scully") //Yes it is!
 
 /datum/holiday/usa
-	name = "Independence Day"
+	name = "US Independence Day"
```
Comment (ATH1909) [inline code/modules/holiday/holidays.dm:256]:
```suggestion
	name = "Independence Day of the United States of America"
```

**Comment by Rohesie** [comment]
Remove or replace with Waitangi day, plz.

**Review by Cyberboss — CHANGES_REQUESTED** [review]
^

**Comment by optimumtact** [comment]
it was a joke you dent heads

**Comment by Rohesie** [comment]
> 
> 
> it was a joke you dent heads

You fail to grasp the seriousness of the situation.

**Comment by optimumtact** [comment]
Grasp this 🥇

**Pull request merged**